src/main/java/com/google/neighborgood/servlets/TaskServlet.java

```diff
@@ -93,8 +96,17 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       }
       out.append("<div class='task-container'>");
       out.append("<div class='task-header'>");
+
+      // Retrieves task's user to get user's nickname
+      String taskUserId = (String) entity.getProperty("userId");
+      Query userQuery =
+          new Query("UserInfo")
+              .setFilter(new FilterPredicate("userId", FilterOperator.EQUAL, taskUserId));
```

Comment:
One thing: Is the `userId` field overlapped with the `Owner` field for the `UserInfo` entity? I don't remember that I've set a userId field before, but now I found that in the `doPost()` of `TaskServlet.java` file, a `userId` field is set for the `UserInfo` entity, but its content is the same as `Owner` field.

Reply:
Ah yes, I think I added at some point when the tasks were hard coded still. At that point Owner was the name/nickname, so they weren’t duplicated - we should definitely remove one of them. I’ll update this in the next commit (I think we’ve been using the “owner” one the most so far, so I’ll remove the userId one).